packages/code-du-travail-frontend/src/outils/publicodes/PubliHandler.tsx

```diff
@@ -0,0 +1,52 @@
+import Engine from "publicodes";
+import { useMemo, useState } from "react";
+
+import { PublicodeContextInterface } from "./PubliContext";
+
+interface State {
+  engine: Partial<Engine>;
+  rule: string;
+}
+
+const usePublicodeHandler = ({
```

Comment:
Est ce qu'on ne devrait pas limiter l'utilisation de use au hooks ? est ce publicodeFinalformAdapter ne serait pas plus approprié ?

Reply:
It's a hook :)